UserTools/PrepareSubSamples/PrepareSubSamples.h

```diff
@@ -0,0 +1,51 @@
+#ifndef PrepareSubSamples_H
+#define PrepareSubSamples_H
+
+#include <string>
+#include <iostream>
+
+#include "Tool.h"
+#include "TimeDelta.h"
+
+class PrepareSubSamples: public Tool {
+
+
+ public:
+
+  PrepareSubSamples();
+  bool Initialise(std::string configfile,DataModel &data);
+  bool Execute();
+  bool Finalise();
+
+
+ private:
+
+  /// The desired maximum SubSample length
+  TimeDelta m_sample_width;
+  /// The desired SubSample overlap time
+  TimeDelta m_sample_overlap;
+
+  int verbose;
+
+  std::stringstream ss;
```

Comment:
```suggestion
  int m_verbose;

  std::stringstream m_ss;
```
At least I've changed the template tool, and some others, since the coding conventions were set...